app/models/member.js

```diff
@@ -4,5 +4,6 @@ import attr from 'ember-data/attr';
 export default DS.Model.extend({
     name: attr('string'),
     email: attr('string'),
-    createdAt: attr('moment-utc')
+    createdAt: attr('moment-utc'),
+    subscriptions: attr('member-subscription')
```

Comment:
@kevinansfield Would be cool if you can take a 👀 if this is the best way to add `subscriptions` info on member model. This uses the transform + separate model definition way which seemed to be the right way from other references in Admin 😄

Reply:
👍 looks fine to me